docs/en/thirdparty/pioarduino.rst

```diff
@@ -0,0 +1,16 @@
+#######################################################
+pioarduino - (p)eople (i)nitiated (o)ptimized (arduino)
+#######################################################
+
+.. warning::
+  Note that this tool is **not maintained by the ESP32 Arduino Core team**, so we cannot provide support or guarantee that it will work as expected.
```

Comment:
```suggestion
  This tool is **not maintained by the ESP32 Arduino Core team**, so we cannot provide support or guarantee that it will work as expected.
```